Dockerfile

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) 2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+FROM node:10.16-stretch-slim
```

Comment:
Would it be feasible to try and use the upstream [.NET Core image](https://hub.docker.com/_/microsoft-dotnet-core) as a base?

Reply:
yep, tried it and it looks better! Even more the image based on upstream .NET Core is smaller on 87

Reply:
Nice :+1: